Modules/Bibliographic/classes/Field/class.ilTable1.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use ILIAS\UI\URLBuilder;
+
+/**
+ * Class ilTable
+ *
+ */
+class ilTable1
+{
+    use \ILIAS\Modules\OrgUnit\ARHelper\DIC;
+    protected array $filter = [];
+    protected ILIAS\UI\Implementation\Component\Table\Data $table;
+    protected \ilBiblAdminFactoryFacadeInterface $facade;
+
+
+    /**
+     * ilTable constructor.
+     */
+
+    public function __construct(?object $a_parent_obj, ilBiblAdminFactoryFacadeInterface $facade)
+    {
+        $this->facade = $facade;
+        $this->parent_obj = $a_parent_obj;
```

Comment:
Das Property ist nicht deklariert, 

```
protected ilBiblAdminFieldGUI $parent_obj;
```

und in konstruktor kann die entsprechende instanz verlangt werden:

```

public function __construct(
        ilBiblAdminFieldGUI $a_parent_obj, 
        ilBiblAdminFactoryFacadeInterface $facade
    ) {
```